### ISSUE TITLE
Minor update to Toggle component on Web

### ISSUE THREAD

**Issue by janogarcia** [vector-im/compound#221]
# Figma

[Compound Web Components · Toggle](https://www.figma.com/file/rTaQE2nIUSLav4Tg3nozq7/Compound-Web-Components?type=design&node-id=119-248&mode=design&t=xC4PKSqudFSJ04Dt-0) 

# Platform

Web

```[tasklist]
- [ ] Change component height from 24 to 20.
- [ ] Change component width from 40 to 36.
```

# Notes

It shouldn't require any extra padding at the component level to comply with the upcoming [upcoming WCAG 2.2 recommendation on minimum target sizes](https://www.w3.org/TR/WCAG22/#target-size-minimum) for AA, as it's extremely unlikely (just bad design that shouldn't pass a design or implementation review) that it won't have at least 2px margin on the top and bottom to account for a 24px circle from the center as described in the guidelines.

**Comment by robintown** [comment]
I've taken this since we'll be needing the toggle component soon in Element Call

**Comment by robintown** [comment]
This issue can be closed now (I apparently don't have permission to do that myself).

**Comment by germain-gg** [comment]
@robintown you should have access to this now! changed the porject permissions
